app/controllers/observations_controller/new_and_create.rb

```diff
@@ -67,9 +67,14 @@ def defaults_from_last_observation_created
     %w[when where location is_collection_location gps_hidden].each do |attr|
       @observation.send("#{attr}=", last_observation.send(attr))
     end
+
     last_observation.projects.each do |project|
-      @project_checks[project.id] = true unless project.open_membership
+      next if project.open_membership ||
```

Comment:
Rationale for different treatment of open vs cosed projects?

Reply:
I prefer the new behavior for open projects and I didn't want to change the behavior for existing projects.  Maybe it should really be related to whether the project has constraints.  If it doesn't have constraints, then it uses what was last set.  If it does have constraints then it is automatically selected when the constraints are currently met.  However, this implies getting the location info before the project selection happens.